source-facebook-marketing/VERSION

```diff
@@ -0,0 +1 @@
+v4
```

Comment:
Every relevant task is already running this code, so I say we overwrite `:v4` and be done with. This isn't really a version upgrade so much as a swap-in-place